elements/pl-number-input/pl-number-input.py

```diff
@@ -7,6 +7,20 @@
 import random
 
 
+RTOL_DEFAULT = 1e-2
+ATOL_DEFAULT = 1e-8
+SIZE_DEFAULT = 35
+NONE_DEFAULT = None
```

Comment:
Not sure if this is necessary, I think `None` is close enough to a constant as it.

Reply:
I agree 😄